### PR TITLE
use async API for spawning FFmpeg

### PR DESCRIPTION
**Pull request by dntsdnts**

### CODE REVIEW

File: export_clip.lua [hunk 1]
```diff
@@ -38,32 +38,36 @@ function export_loop_clip()
...
-      '-t', tostring(b-a),
-      '-c:v', 'libx264',
-      '-crf', '22',
-      '-pix_fmt', 'yuv420p',
-      '-filter_complex', 'scale=iw*min(1\\,min(1280/iw\\,720/ih)):-2',
-      '-an', '-sn',
-      '-map_metadata', '-1',
-      -- '-map', '0',
-      '-v', 'error',
+      "run",
```
Comment (citrus-lemon) [inline export_clip.lua:44]: what is this "run" command

File: export_clip.lua [hunk 2]
```diff
@@ -38,32 +38,36 @@ function export_loop_clip()
...
+      "-t", tostring(b-a),
+      "-c:v", "libx264",
+      "-crf", "21",
+      "-preset:v", "placebo",
+      "-pix_fmt", "yuva420p",
+      "-filter_complex", "scale=iw*min(1\\,min(1280/iw\\,720/ih)):-2",
+      "-an", "-sn",
+      "-map_metadata", "-1",
+      "-v", "error",
+      "-n",
```
Comment (citrus-lemon) [inline export_clip.lua:57]: why use double quote instead of single quote

File: export_clip.lua (hunk 1 above)
Comment (dntsdnts) [reply export_clip.lua:44]: https://mpv.io/manual/stable/#command-interface-[-]]]

**Comment by citrus-lemon** [comment]
merged on https://github.com/WindProphet/mpv-export-clip/commit/86a05bb7f661839f66f71a490a9cc83a0f88f42d

**Pull request closed without merging**